### PR TITLE
chore: delay linea bridge tx to make it less flaky

### PR DESCRIPTION
**Pull request by infiniteflower**
## **Description**

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/29109?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Set network to Linea
2. Bridge an ERC20 to another network
3. Observe loading spinner on quotes page
4. Get redirected to activity list
5. See successful ERC20 approval and bridge tx 

### **After**

https://github.com/user-attachments/assets/e6079f43-632e-4bd3-a9e0-54f7c427b541

## **Pre-merge author checklist**

- [x] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by jclancy93 — APPROVED** [review]
LGTM

Not blocking: we may want to consider creating another ticket to change the button text to add some text like "approving..." beside the spinner

**Review by ghgoodreau — APPROVED** [review]

File: ui/pages/bridge/prepare/bridge-cta-button.tsx [hunk 1]
```diff
@@ -108,20 +109,29 @@ export const BridgeCTAButton = () => {
...
+                  ...tradeProperties,
+                },
+              });
+            submitBridgeTransaction(activeQuote);
+          } catch (error) {
+            setIsSubmitting(false);
+          }
         }
       }}
+      loading={isSubmitting}
```
Comment (micaelae) [inline ui/pages/bridge/prepare/bridge-cta-button.tsx:134]: Is the button disabled when it's loading?

File: ui/pages/bridge/prepare/bridge-cta-button.tsx [hunk 2]
```diff
@@ -108,20 +109,29 @@ export const BridgeCTAButton = () => {
...
+                properties: {
+                  ...quoteRequestProperties,
+                  ...requestMetadataProperties,
+                  ...tradeProperties,
+                },
+              });
+            submitBridgeTransaction(activeQuote);
+          } catch (error) {
+            setIsSubmitting(false);
+          }
```
Comment (micaelae) [inline ui/pages/bridge/prepare/bridge-cta-button.tsx:131]: Can we use `finally` instead?

File: ui/pages/bridge/prepare/bridge-cta-button.tsx (hunk 1 above)
Comment (infiniteflower) [reply ui/pages/bridge/prepare/bridge-cta-button.tsx:134]: Handled in https://github.com/MetaMask/metamask-extension/pull/29149

File: ui/pages/bridge/prepare/bridge-cta-button.tsx (hunk 2 above)
Comment (infiniteflower) [reply ui/pages/bridge/prepare/bridge-cta-button.tsx:131]: Handled in https://github.com/MetaMask/metamask-extension/pull/29149

**Pull request merged**